tests/foreman/cli/test_discoveryrule.py

```diff
@@ -255,13 +261,12 @@ def test_negative_create_with_invalid_name(self):
 
         :CaseLevel: Component
         """
-        for name in invalid_values_list():
-            with self.subTest(name):
-                with self.assertRaises(CLIFactoryError):
-                    self._make_discoveryrule({'name': name})
+        with pytest.raises(CLIFactoryError):
+            discoveryrule_factory(options={'name': name})
 
     @pytest.mark.tier3
-    def test_negative_create_with_invalid_hostname(self):
+    @pytest.mark.parametrize('name', **parametrized(invalid_hostnames_list()))
```

Comment:
Hard to see context in mobile view, but do these tests have the parameterized metadata?

Reply:
UHHH YEAH TOTALLY.  As many times as I've made this comment, I forget it myself. Will add that to the commit ASAP.

Reply:
That's unfortunate that i wasn't able to see that added in mobile view ;)